L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -33,3 +33,7 @@
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
 
+# this is to run Tracklet pattern reco with new KF
+TrackletTracksFromTrackletEmulation = TTTracksFromTrackletEmulation.clone(
+                                               Fakefit = cms.bool(True)
```

Comment:
Is cfg parameter Fakefit actually used anywhere in the C++?

Reply:
Has been used in the past and is important. I don't see it been used anymore. I guess the usage has been lost during merges.

Reply:
There it is:
https://github.com/cms-L1TK/cmssw/blob/tschuh/L1Trigger/TrackFindingTracklet/plugins/L1FPGATrackProducer.cc#L266

Reply:
I added the lost line again, that impacts tracking performance, you might want to redo your last tests.